### DISCUSSION TITLE
How to run this offline?

### DISCUSSION THREAD

**Discussion by sameeryel** [dd3v/snippets.ninja#1] [Q&A] [answered]
I am new to github and snippets.ninja. How do we run this offline?

**Comment by dd3v** [comment, marked as answer]
@sameeryel hello! When the app will be installed it will be available in offline mode.

<img width="1241" alt="Screenshot 2022-12-19 at 14 51 54" src="https://user-images.githubusercontent.com/2827559/208440876-a57ff19b-ae30-4384-8236-f7d9f2609a71.png">
